lib/report.js

```diff
@@ -16,6 +16,7 @@ const getSourceMapFromFile = require('./source-map-from-file')
 const v8toIstanbul = require('v8-to-istanbul')
 const util = require('util')
 const debuglog = util.debuglog('c8')
+const { CoverageReport } = require('monocart-coverage-reports')
```

Comment:
we should move this into a try/catch.

Reply:
Perhaps we should move if into a try/catch in a separate monocart.js helper file, then we can mock that file to test the help output.